README.md

```diff
@@ -12,30 +12,23 @@ To report an issue using the **Language Forge** service, email "languageforgeiss
 
 ## Special Thanks To ##
 
-![BrowserStack Logo](readme_images/browserstack-logo.png "BrowserStack") for end-to-end test automation.
-
```

Comment:
Let's leave this in the README.md as we still have a registered OSS license for Browser Stack and are still using it.  I will make a separate GH suggestion above to add it back in.   We aren't using it for e2e testing anymore but are actively using it for device testing.

Reply:
thanks, I did not realize that.